app/models/events/search.rb

```diff
@@ -76,7 +76,7 @@ def query_events_api(limit)
     end
 
     def start_of_month
-      return nil if month.blank?
+      return DateTime.now.beginning_of_day if month.blank?
```

Comment:
Does handle timezone? Should this be `Time.zone.now.beginnging_of_day`

Or should it be the start of day for the users timezone?

Reply:
Good catch, the API expects UTC - I'll update that now.

Reply:
:+1: